front/pages/w/[wId]/builder/assistants/create.tsx

```diff
@@ -179,25 +166,6 @@ export default function CreateAssistant({
     }, 500);
   };
 
-  useEffect(() => {
```

Comment:
The issue by removing this, is that we won't be able to share link for templates. Example: https://dust.tt/w/0ec9852c2f/builder/assistants/create?flow=workspace_assistants&templateId=d1abfbae81.

What's the rational to remove it?

Reply:
It still works with what I did: https://front-edge.dust.tt/w/0ec9852c2f/builder/assistants/create?flow=workspace_assistants&templateId=d1abfbae81

Reply:
(my branch is on front-edge of course 😄 )

Reply:
Looking.

Reply:
👀

Reply:
It does work @flvndvd, the link I shared does open the template modal. What exactly is not working?

Reply:
I'm not sure we are talking about the same thing then 🙃. I was just saying that it's really convenient to have the template Id included in the URL. But with your PR (unless I missed something), it looks like the template ID isn't in the URL anymore, which makes it hard to share. Does that make sense?

Reply:
Ok got it, added a commit 🙏🏻